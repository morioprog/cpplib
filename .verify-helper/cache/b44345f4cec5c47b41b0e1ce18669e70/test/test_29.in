9
-5 4
-8 1
-7 8
-9 7
1 8
-4 7
-8 9
7 6
0 9
