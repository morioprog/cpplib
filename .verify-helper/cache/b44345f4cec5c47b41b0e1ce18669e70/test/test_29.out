7 6
1 8
0 9
-4 7
-7 8
-8 9
-5 4
-9 7
-8 1
